scss/4-elements/_links.scss

```diff
@@ -22,12 +27,13 @@ a,
   }
 }
 
-.cads-linkbutton {
+button.cads-linkbutton {
+  // Reset button styles when styled like a link link
+  -webkit-appearance: none;
   border: none;
   background: transparent;
   padding: 0;
   box-shadow: none;
 
-  @extend .cads-heading-small;
```

Comment:
`.cads-heading-small` was being almost 100% overrides by `. cads-hyperlink`. Another reason why extends are tricksy. Decoupled this from the main button styles completely as it was overriding so much already.